.github/workflows/cypress.yml

```diff
@@ -0,0 +1,36 @@
+name: E2E Tests
+
+on: [push]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Prepare repository
+        run: git fetch --unshallow --tags
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: yarn-deps-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            yarn-deps-${{ hashFiles('yarn.lock') }}
+      - run: yarn install --frozen-lockfile
+
+      - name: Run e2e tests
+        uses: cypress-io/github-action@v2
+        with:
+          build: yarn build
+          start: yarn e2e:ci
+        env:
```

Comment:
On the CircleCI job, there is a `CYPRESS_CACHE_FOLDER` env, don't we need it here as well?

Reply:
Yes I remember something about caching the install of Cypress itself (besides caching the usual node_modules), so far it didn't cause an issue. But I'll take a bit of time to read on it to be sure if it is needed in this case or not.

Reply:
So if i understand correctly `cypress-io/github-action@v2` already does it. 

However, if we want to cache node_modules independently, we can use the following example https://github.com/bahmutov/cypress-gh-action-split-install/blob/master/.github/workflows/tests.yml 

so unless we want to do advanced parallel runs and caching, it should be fine as is.